src/main/java/com/example/bookshop/repository/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.example.bookshop.repository;
+
+import com.example.bookshop.model.Book;
+import java.util.List;
+
+public interface BookRepository {
```

Comment:
try to use JpaRepository, so you will no longer need to create an implementation